core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/RootResource.java

```diff
@@ -43,6 +43,7 @@ public class RootResource {
   @Operation(summary = "Schema Registry Root Resource",
       description = "The Root resource is a no-op.",
       responses = {@ApiResponse(content = @Content(schema = @Schema(implementation = Map.class)))})
+  @DocumentedName("rootResourceGet")
```

Comment:
Should this be getRootResource? It will follow a similar pattern with other names [here](https://confluentinc.atlassian.net/wiki/spaces/~44042347/pages/2799440124/Documented+Names+for+SR+APIs)